data/research.json

```diff
@@ -0,0 +1,87 @@
+        "linkUrl": "https://eprint.iacr.org/2020/1478.pdf"
+    },
+    {
+        "title": "An Offline Delegatable Cryptocurrency System",
+        "imageSrc": "offline-crypto-system.jpg",
+        "authors": "Rujia Li, Qin Wang, Xinrui Zhang, Qi Wang, David Galindo and Yang Xiang",
+        "desc": "We propose DelegaCoin, an offline delegatable cryptocurrency system that uses trusted execution environments (TEEs) as decentralized \"virtual agents\" to prevent malicious delegation. In DelegaCoin, an owner can delegate his coins through offline-transactions without interacting with the blockchain network.",
+        "linkUrl": "https://ieeexplore.ieee.org/document/9461112"
+    },
+    {
+        "title": "A Practical Framework for General Dialogue-based Bilateral Interactions",
+        "imageSrc": "practical-framework.jpg",
+        "authors": "Seyed Ali Hosseini, David Minarsch, Marco Favorito",
+        "desc": "For autonomous agents and services to cooperate and interact in multiagent environments they require well-defined protocols. A multitude of protocol languages for multi-agent systems have been proposed in the past, but they have mostly remained theoretical or have limited prototypical implementations.",
+        "linkUrl": "https://link.springer.com/chapter/10.1007/978-3-030-97457-2_18"
+    },
+    {
+        "title": "Symbolic modelling of remote attestation protocols for device and app integrity on Android",
+        "imageSrc": "symbolic-modelling-on-android.jpg",
+        "authors": "Abdulla Aldoseri, Tom Chothia, José Moreira, David F. Oswald",
+        "linkUrl": "https://dl.acm.org/doi/10.1145/3579856.3582812"
+    },
+    {
+        "title": "Automatic verification of transparency protocols",
+        "imageSrc": "automatic-verification.jpg",
+        "authors": "Vincent Cheval, José Moreira, Mark Ryan",
+        "linkUrl": "https://ieeexplore.ieee.org/document/10190509"
+    },
+    {
+        "title": "Protocols for a Two-Tiered Trusted Computing Base",
+        "imageSrc": "protocols-for-2-tiered.jpg",
+        "authors": "José Moreira, Mark Ryan, Flavio D. Garcia",
+        "linkUrl": "https://link.springer.com/chapter/10.1007/978-3-031-17143-7_12"
+    },
+    {
+        "title": "Transparent Registration-Based Encryption through Blockchain",
+        "imageSrc": "transparent-reg-based-encryption.jpg",
+        "authors": "Qin Wang, Rujia Li, Qi Wang, David Galindo, Shiping Chen, Yang Xiang",
+        "linkUrl": "https://dl.acm.org/doi/10.1145/3568315"
+    },
+    {
+        "title": "SoK: TEE-Assisted Confidential Smart Contract",
+        "imageSrc": "SoK.jpg",
+        "authors": "Rujia Li, Qin Wang, Qi Wang, David Galindo, Mark Ryan",
+        "linkUrl": "https://petsymposium.org/popets/2022/popets-2022-0093.php"
+    },
+    {
+        "title": "Robust Subgroup Multi-Signatures for Consensus",
```

Comment:
have you asked anyone about this duplicate?

Reply:
Yes, have asked Vibhuti and she said to keep both :)